string file
bool pub_octo
---
bool result